app/components/collections/show/settings_header_component.rb

```diff
@@ -16,7 +16,7 @@ def initialize(collection_version:)
       delegate :depositing?, :draft?, :collection, to: :collection_version
 
       def name
-        collection_version.name.presence || 'No Title'
+        DepositTitlePresenter.show(collection_version)
       end
```

Comment:
refactored the existing `WorkTitlePresenter` component so we can pass it a collection too (see below) and then used that throughout